saviSimulation/savi/asl/uav.asl

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Does the UAV ever climb? I think it should at the very least it should climb to some altitude early in the program, otherwise I don't think it is actually airborne.

@gtrabes : a suggestion for the future could be that UAVs can only move if they have an altitude greater than 0.

Reply:
We can implement that

Reply:
I'm going to open an issue for this. The UAV needs to be airborne to move. This will require simulation side and behaviour side changes.